### DISCUSSION TITLE
Stellarium problem

### DISCUSSION THREAD

**Discussion by 1963m** [Stellarium/stellarium#3454] [General]
I have a problem with Stellarium on Windows 11, I can't get the telescope to advance, it is also not known which version of Stellarium is the correct one for Windows 11, if it is Qt5 or Qt6, there is no information

**Comment by gzotti** [comment]
No information? And Google does not find anything relevant about "Telescope control Stellarium"? 

From the Release Notes:

23.1:

> TELESCOPE USERS
> 
> Like in the 1.* series, it seems the TelescopeControl plugin at least on Windows works better in the Qt5-based releases. Just install this if you are affected. We are looking for telescope users with programming skills to help us fixing this issue. Reward possible!
> 

23.2:

> TELESCOPE USERS
> 
> Like in the 1.* series, it seems the TelescopeControl plugin at least on Windows works better in the Qt5-based releases. Just install this if you are affected. We are looking for telescope users with programming skills to help us fixing this issue. Reward possible!

23.3:

> TELESCOPE USERS
> 
> We have identified and fixed some bugs in the TelescopeControl plugin on the Qt6-based releases.
> 
> Telescope users, please try whether this solves your problems. If not, we are looking for those of you with programming skills to help us fixing remaining issues. Reward possible!
>
